docs/pages/eas-update/error-recovery.mdx

```diff
@@ -19,7 +19,7 @@ The important thing is to **publish a new update with a fix as soon as possible
 
 The first thing to try is rolling back to an older update that you know was working. **However, this may not always be safe;** your broken update may, for example, have modified persistent state (such as data stored in AsyncStorage or on the device's file system) in a non-backwards-compatible way. It's important to test in a staging environment that emulates an end user's device state as closely as possible to load the broken update and then roll back.
 
-If you can identify an older update that is safe to roll back to, you can do so using [EAS Update's `republish` option](/eas-update/eas-cli/#republish-a-previous-update-within-a-branch).
+If you can identify an older update that is safe to roll back to, you can do so using EAS Update's `republish` option from the [website](https://expo.dev/accounts/[account]/projects/[project]/updates) or [CLI](/eas-update/eas-cli/#republish-a-previous-update-within-a-branch).
```

Comment:
```suggestion
If you can identify an older update that is safe to roll back to, you can do so using EAS Update's `republish` option from [Expo dashboard](https://expo.dev/accounts/[account]/projects/[project]/updates) or [EAS CLI](/eas-update/eas-cli/#republish-a-previous-update-within-a-branch).
```

I think instead of mentioning website, we can be specific here and use Expo dashboard. Similarly, let's also use EAS CLI.